### DISCUSSION TITLE
EAP

### DISCUSSION THREAD

**Discussion by lancemao** [ctran/TinyRadius#48] [General]
Hey, any plan to support for eap?

**Comment by ctran** [comment]
There's no plan to add any features from me.  You're open to contribute :-)

**Comment by lancemao** [comment]
I'd love to. 

We are investing time and effort for EAP. We are looking into freeradius as it claims to be the only open source RADIUS server that supports EAP
I have been googling for couple of days, also read rfc3579 several times. I got NO CLUE how to implement it.

according to this post:
https://stackoverflow.com/questions/19097125/how-and-where-radius-and-eap-combine

the flow is clear, the problem is that there seems no information about the implementation detail

If by any chance, you have some article / book/ guidance, I'd love to implement EAP on top of TinyRadius. Please contact me by:
maolongdong@gmail.com

**Reply by weijie0** [reply]
Did you solved it
